scripts/enabled_test_modules.py

```diff
@@ -457,7 +457,7 @@
         'No overload variant of "join" matches argument types "str", "None"',
         'Argument 1 to "Archive" has incompatible type "None"; expected "str"',
         'Argument 1 to "to_path" has incompatible type "int"; expected "Union[Path, str]"',
-
+        'Cannot infer type argument 1 of "cached_property"',
```

Comment:
Caused by untyped `lambda` used in Django tests (tests/utils_tests/test_functional.py:177):
```python
    def test_cached_property_set_name_not_called(self):
        cp = cached_property(lambda s: None)
```